dimod/sampleset.py

```diff
@@ -1257,23 +1298,23 @@ def slice(self, *slice_args, **kwargs):
             8  0  0  0  0  0  0  0  0  1  0      8       1
             9  0  0  0  0  0  0  0  0  0  1      9       1
             ['BINARY', 10 rows, 10 samples, 10 variables]
-
+            >>>
```

Comment:
Why add the carrots?

Reply:
Keeps the whole thing in one example in RTD, with a single scroll bar, as opposed to partial examples, each with a scroll bar. (And doctest broke when I had ellipses.)

Reply:
Probably it would be better to just shorten the width then. E.g.
```
            >>> import numpy as np
            ...
            >>> sampleset = dimod.SampleSet.from_samples(np.diag(range(1, 11)),
            ...                                                                              dimod.BINARY, energy=range(10))
            >>> print(sampleset)
               0  1  2  3  4  5  6  7  8  9 energy num_oc.
            0  1  0  0  0  0  0  0  0  0  0      0       1
            1  0  1  0  0  0  0  0  0  0  0      1       1
            2  0  0  1  0  0  0  0  0  0  0      2       1
            3  0  0  0  1  0  0  0  0  0  0      3       1
            4  0  0  0  0  1  0  0  0  0  0      4       1
            5  0  0  0  0  0  1  0  0  0  0      5       1
            6  0  0  0  0  0  0  1  0  0  0      6       1
            7  0  0  0  0  0  0  0  1  0  0      7       1
            8  0  0  0  0  0  0  0  0  1  0      8       1
            9  0  0  0  0  0  0  0  0  0  1      9       1
            ['BINARY', 10 rows, 10 samples, 10 variables]
```

Reply:
That will get rid of the scrolls but will keep the separation into partial examples

Reply:
I like the partial examples personally, but ymmv.

Reply:
Our other examples are self-contained so I'd go with consistency barring some improvement to clarity. 
"ymmv": my daily new acronym

Reply:
@JoelPasvolsky, I vote for no prompts between these examples. It's much nicer to have them self-contained. If you don't like the series of boxes on output, you can extract the comment line as a one line description of the example.

Reply:
Okay, looks like we have mob rule in sampleset. I split it without the comment lines:
![image](https://user-images.githubusercontent.com/34041130/61297833-eb392c80-a791-11e9-88d4-6aa0f5581632.png)